src/google/protobuf/compiler/cpp/cpp_map_field.cc

```diff
@@ -229,7 +229,10 @@ void MapFieldGenerator::GenerateSerializeWithCachedSizesToArray(
   if (utf8_check) {
     format(
         "struct Utf8Check {\n"
-        "  static void Check(ConstPtr p) {\n");
+        "  static void Check(ConstPtr p) {\n"
+		// p may be unused when GetUtf8CheckMode evaluates to kNone,
```

Comment:
Looks good but would you mind fixing the indentation? (I think your editor must have inserted tabs whereas we always use spaces.)

Reply:
Should be fixed now.

Btw, have you considered providing [.editorconfig](https://editorconfig.org/) with the basic indentation rules?

Reply:
The formatting doesn't look quite right yet, maybe you just need to do a git push?

I hadn't heard of editorconfig before. I wonder if someone has already written a config to match the Google C++ style guide.

Reply:
Sorry, something went wrong with git pushing.
Should be fixed now.